### PR TITLE
fix #83: Merge keywords and default values for aliases

### LINKED ISSUE #83: Default value and keywords of exported aliases not merged
It would be nice if we could get the default values and keywords of exported aliases

```js
    /**
     * CSS class for the component
     * @type {'main' | 'sidebar'}
     * 
     */
    let klass = "main";

    export {klass as class}
```

Current output:
```js
    {
      "visibility": "public",
      "description": "CSS class for the component",
      "keywords": [],
      ...
      "type": {
        "kind": "union",
        "text": "'main' | 'sidebar'",
        "type": [
          {
            ...
          },
          {
            ...
          }
        ]
      },
      "localName": "klass"
    }
```

Preferred output:
```js
    {
      "keywords": [
        {
          "name": "type",
          "description": "{'main' | 'sidebar'}"
        }
      ],
      "visibility": "public",
      "description": "CSS class for the component",
      ...
      "type": {
        "kind": "union",
        "text": "'main' | 'sidebar'",
        "type": [...] //same as above
      },
      "localName": "klass",
      "defaultValue": "main"
    }
```

### PR DESCRIPTION
**Pull request by ekhaled**
closes #83

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,13 +77,17 @@ function mergeItems(itemType, currentItem, newItem, ignoreLocations) {
...
+    }
+
     if (!currentItem.type || currentItem.type.type === 'any') {
         if (newItem.type && newItem.type.type !== 'any') {
             currentItem.type = newItem.type;
         }
     }
 
-    if (!currentItem.keywords && newItem.keywords) {
+    if ((!currentItem.keywords || currentItem.keywords.length == 0) && newItem.keywords) {
```
Comment (alexprey) [inline index.js:90]: Should we override keywords or may we can merge that?

**Review by alexprey — COMMENTED** [review]
Thanks for your fix! Sorry for my delay on review

File: index.js (hunk 1 above)
Comment (ekhaled) [reply index.js:90]: I don't think keywords are even picked up from alias declarations.

Comment (alexprey) [reply index.js:90]: Ok, thanks!

**Pull request merged**